def f():
    a: int = 'x'   # Maybe this annotation should be an error?
